engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -342,11 +342,16 @@ class WorkflowManagerActor(config: Config)
 
     import PromiseActor.EnhancedActorRef
 
+    val shadowExecutionEnabled = config.getBoolean("system.shadowExecutionEnabled")
+
     val message  = MaterializeWorkflowDescriptorActor.MaterializeWorkflow(workflowId, source, config)
     val materializeWorkflowDescriptorActor = context.actorOf(MaterializeWorkflowDescriptorActor.props())
     materializeWorkflowDescriptorActor.askNoTimeout(message) map {
       case MaterializationSuccess(descriptor) =>
-        val wfActor = context.actorOf(WorkflowActor.props(descriptor), s"WorkflowActor-$workflowId")
+        val wfActor = shadowExecutionEnabled match {
+          case true => context.actorOf(ShadowWorkflowActor.props(), s"WorkflowActor-$workflowId")
+          case _ => context.actorOf(WorkflowActor.props(descriptor), s"WorkflowActor-$workflowId")
+        }
```

Comment:
I would have thought that having them operate totally different FSMs is expected and OK? Ideally, I would only want to change OldeStyleeWorkflowActor if tests fail or it won't compile.

IIRC, this descriptor is actually going to be `WdlSources` for the new actor?